### PR TITLE
Grammar enhancement

### PR DESCRIPTION
**Pull request by Fractal-Tess**
Hey there,

While I was using this template, I noticed a few grammar mistakes and took the time to correct them.
I also added the `Permissions` type to `security-restrictions.ts` for better readability.

If you don't like anything I changed, feel free to revert it or reject this PR altogether.
Also, make sure I did not brick anything, as there were new commits on the remote and I had to do my first rebase every. 

Thank you for your time!

### CODE REVIEW

File: packages/main/src/mainWindow.ts [hunk 1]
```diff
@@ -4,18 +4,23 @@ import {URL} from 'url';
 
 async function createWindow() {
   const browserWindow = new BrowserWindow({
-    show: false, // Use 'ready-to-show' event to show window
+    show: false, // Use the 'ready-to-show' event to show the instantiated BrowserWindow.
     webPreferences: {
-      webviewTag: false, // The webview tag is not recommended. Consider alternatives like iframe or Electron's BrowserView. https://www.electronjs.org/docs/latest/api/webview-tag#warning
+      // The webview tag is not recommended.
+      // Consider alternatives like an iframe or Electron's BrowserView.
+      // @see https://www.electronjs.org/docs/latest/api/webview-tag#warning
```
Comment (cawa-93) [inline packages/main/src/mainWindow.ts:11]: Pls, use JSDoc here

**Review by cawa-93 — CHANGES_REQUESTED** [review]

**Comment by cawa-93** [comment]
Huge thanks

File: packages/main/src/mainWindow.ts (hunk 1 above)
Comment (Fractal-Tess) [reply packages/main/src/mainWindow.ts]: I reverted to the old comment

**Comment by cawa-93** [comment]
Awesome work

**Pull request merged**